app/views/layouts/application.html.erb

```diff
@@ -23,13 +23,6 @@
 </head>
 
 <body>
-
-
-
-  <div devise notices and allerts>
-    <p class="notice"><%= notice %></p>
-    <p class="alert"><%= alert %></p>
-  </div>
```

Comment:
Why were these removed? Do we not need devise alerts/notices anymore? I know you've implemented a notification system, but is devise properly routed through the custom notification model?

Reply:
That wasn't removed, it was moved to a bit below in the same file